subiquity/server/server.py

```diff
@@ -178,6 +178,21 @@ async def free_only_POST(self, enable: bool) -> None:
         # enabling free only mode means disabling components
         self.app.base_model.mirror.disable_components(to_disable, enable)
 
+    async def interactive_sections_GET(self) -> Optional[List[str]]:
+        if self.app.autoinstall_config is None:
```

Comment:
One more comment (sorry), which of returning None here vs all sections do you think is better?

Reply:
I like `None` better, actually.  It more closely reflects the way the autoinstall file was written - that no interactive-sections was supplied.